main.go

```diff
@@ -110,13 +112,45 @@ func (u *urlsStruct) home(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func urlsFromJSON(u *urlsStruct, f string) {
+	data, err := ioutil.ReadFile(f)
+	if err != nil {
+		log.Fatalf("Read json file failed")
+	}
+
+	var result map[string]string
+
+	err = json.Unmarshal([]byte(data), &result)
+	if err != nil {
+		log.Fatalf("Error on decode json")
+	}
+
+	for key, el := range result {
+		fmt.Println(key, el)
+		u.mux.Lock()
+		u.urls[key] = el
+		atomic.AddInt32(&u.Stats.UrlsGenerated, 1)
+		u.mux.Unlock()
+	}
+}
+
 func main() {
+	var serverAddr string
+	var jsonPath string
```

Comment:
style:
Generally to declare variable it's easier and more readable to just use `serverAddr := ""`.
Not using the `var` keyword is preferred, unless in some rare cases when you want to explicitly show the reader that letting the variable to its default value is important.